src/Core/src/ImageSources/iOS/ImageSourceExtensions.cs

```diff
@@ -43,5 +45,15 @@ public static partial class ImageSourceExtensions
 						? UIImage.FromFile(filename)
 						: UIImage.FromBundle(filename);
 		}
+
+		internal static Task<IImageSourceServiceResult<UIImage>?> GetPlatformImageAsync(this IImageSource? imageSource, IImageSourceServiceProvider provider, UIKit.UIView uiContext, CancellationToken token = default)
```

Comment:
Review if we need this overload. We might be able to just reuse the existing one to get the density off the MauiContext